frontend/src/citizen-frontend/calendar/CalendarGridView.tsx

```diff
@@ -244,6 +242,10 @@ const PageHeaderRow = styled(ContentArea).attrs({ opaque: false })`
   display: flex;
   justify-content: space-between;
   align-items: center;
+  & > div {
+    display: flex;
+    gap: ${defaultMargins.L};
+  }
```

Comment:
This is quite fragile. I'd convert the container `<div>` to a styled component or just replace it with `<FixedSpaceColumn spacing="L">`